sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/Metrics/ExecutableLinesWalkerTest.cs

```diff
@@ -131,13 +132,15 @@ void Foo(int? i, string s)
                     }
                     var x = s?.Length; // +1
                     var xx = i == 1 ? 1 : 1; // +1
-                }");
+                }",
+              4, 5, 6, 11, 13, 14
+              );

```

Comment:
[cosmetic] you put the closing parenthesis at the end of the number list for the other tests.